tests/python/contrib/test_hexagon/test_benchmark_elemwise_add.py

```diff
@@ -432,3 +422,7 @@ def test_elemwise_add(hexagon_launcher: HexagonLauncherRPC):
 
     if _BT.has_fail() > 0:
         pytest.fail("At least one benchmark configuration failed", pytrace=False)
+
+
+if __name__ == "__main__":
+    sys.exit(tvm.testing.main())
```

Comment:
please change it to `tvm.testing.main()`

Reply:
> please change it to `tvm.testing.main()`

Has this already been addressed?  AFAICT the code is as requested.